xml/System/AppDomain.xml

```diff
@@ -5804,9 +5803,8 @@ This method overload uses the <xref:System.AppDomainSetup> information from the
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The friendly name of the default application domain is the file name of the process executable. For example, if the executable used to start the process is `"c:\MyAppDirectory\MyAssembly.exe"`, the friendly name of the default application domain is `"MyAssembly.exe"`.
-
 
+The friendly name of the default application domain is the file name of the process executable. For example, if the executable used to start the process is `"c:\MyAppDirectory\MyAssembly.exe"`, the friendly name of the default application domain is `"MyAssembly.exe"`. (In .NET (Core) versions, the friendly name doesn't include the file extension.)
```

Comment:
Should this call out .NET 5 + as well as .NET Core?

Reply:
```suggestion
The friendly name of the default application domain is the file name of the process executable. For example, if the executable used to start the process is `"c:\MyAppDirectory\MyAssembly.exe"`, the friendly name of the default application domain is `"MyAssembly.exe"`. (In .NET Core and .NET 5+, the friendly name doesn't include the file extension.)
```